src/notifications/AnnouncementProvider/index.js

```diff
@@ -85,7 +85,7 @@ export const AnnouncementProvider = ({
         updateTime: Date.now(),
       };
     },
-    [context, onNewAnnouncement]
+    [context, onNewAnnouncement, fetchApi]
```

Comment:
why is this in this PR?

Reply:
becuase you asked me to fix the lint iissues

Reply:
it's ok